### ISSUE TITLE
nOCR insists on breaking italic 'r', ':' into two characters

### ISSUE THREAD

**Issue by tarminu** [SubtitleEdit/subtitleedit#5894]
Performing nOCR on a VobSub file that contains italic sequences.

Some characters are being interpreted as two adjacent characters.  The dialogue box to specify a single character for OCR is not being displayed -- if it were, I could expand selection and fix the issue.  Instead it guesses, then shows a dialogue to correct the misspelled words.

I can "Edit image db" from that dialogue, but that only presents me with the two pieces as individual characters and I do not see a way to specify the two should be treated as a single character, nor do I see a way to trigger the dialogue that would allow me to expand the selection.

How do I remedy this?  The movie in question has extensive sequences in italic, and it uses 'r' quite a lot.  This is making it very difficult to proceed with the OCR.

The other OCR options do not perform well for me which is why I am using nOCR.

To summarize:

Is there a way to instruct nOCR to combine two adjacent characters into one what can be triggered when needed?

**Comment by niksedk** [comment]
It really hard to say without the actual subtitle... perhaps you can double click (inspect compare matches) on the line in the list view, edit/expand the `r` character.

**Comment by niksedk** [comment]
Also, the help page for nOCR is here: https://www.nikse.dk/subtitleedit/nocr
